Sources/XcodeGenKit/ProjectGenerator.swift

```diff
@@ -13,10 +13,11 @@ public class ProjectGenerator {
         self.project = project
     }
 
-    public func generateXcodeProject() throws -> XcodeProj {
+    public func generateXcodeProject(to projectDirectory: Path? = nil) throws -> XcodeProj {
```

Comment:
minor nitpick if using naming like that:
```suggestion
    public func generateXcodeProject(in projectDirectory: Path? = nil) throws -> XcodeProj {
```